tools/summaries/build_template_summaries.py

```diff
@@ -165,26 +169,77 @@ def build_template_summaries(place_dcids: List[str], stat_var_json=str) -> Dict:
   return summaries
 
 
-def main():
-  with open("priority-places.json") as f:
-    priority_places = json.load(f)
-
-  with open("places-countries.json") as f:
-    countries = json.load(f)
-
-  priority_summaries = build_template_summaries(place_dcids=priority_places,
-                                                stat_var_json=_STAT_VAR_JSON)
-
-  country_summaries = build_template_summaries(place_dcids=countries,
-                                               stat_var_json=_STAT_VAR_JSON)
-
-  all_summaries = utils.combine_summaries(
-      [priority_summaries, country_summaries])
-
-  utils.write_summaries_to_file(summaries=all_summaries,
-                                output_file=_OUTPUT_FILENAME)
+def build_template_summaries_for_sitemap(
+    sitemap_url: str,
+    stat_var_json: str = _STAT_VAR_JSON,
+    batch_size: int = _BATCH_SIZE,
+    output_file: str = _OUTPUT_FILE) -> Dict:
+  """Generate summaries for all places in a sitemap"""
+  start_time = time.time()
+
+  # Extract places to create summaries for from sitemap
+  places = utils.get_places_from_sitemap(sitemap_url)
+
+  # Get summaries in batched calls
+  batch_num = 0
+  batch_start_time = time.time()
+  for batch in utils.batched(places, batch_size):
+    logging.info(f'Processing batch number {batch_num}')
```

Comment:
Might be helpful to add some more log messages with additional context like:

(1) how many total places we're processing
(2) how many batches there are in total

Reply:
Added log messages with total places and how many batches.